deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -237,7 +393,16 @@ services:
       SETTINGS_HTTP_ADDR: 0.0.0.0:9190
       SETTINGS_GRPC_ADDR: 0.0.0.0:9191
 
-      SETTINGS_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
+      SETTINGS_JWT_SECRET: ${OCIS_JWT_SECRET}
+
+      OCIS_SYSTEM_USER_API_KEY: ${SYSTEM_USER_API_KEY}
+      OCIS_SYSTEM_USER_ID: ${SYSTEM_USER_ID}
+
+      SETTINGS_ADMIN_USER_ID: ${ADMIN_USER_ID}
+
+      STORAGE_GATEWAY_GRPC_ADDR: storage-system:9215
+      STORAGE_GRPC_ADDR: storage-system:9215
+
     volumes:
       - "ocis-settings:/var/lib/ocis"
```

Comment:
the settings extension uses the system-storage in the default configuration and should therefore not need a volume